astro/src/content/docs/apis/_scope-request-body.mdx

```diff
@@ -0,0 +1,30 @@
+import APIBlock from 'src/components/api/APIBlock.astro';
+import APIField from 'src/components/api/APIField.astro';
+import JSON from 'src/components/JSON.astro';
+
+#### Request Body
+
+<APIBlock>
+  <APIField name="scope.data" type="Object" optional>
+    An object that can hold any information about the OAuth Scope that should be persisted.
+  </APIField>
```

Comment:
That's right, we are treating custom scopes as implicitly enabled once created. Although, giving an admin the ability to  disable a custom scope without having to delete it might be a nice feature to add later after getting some feedback from early adopters.

Reply:
Is that implicit 'enabling' documented anywhere? If not, can you please add that to the scopes page?

Reply:
For sure. Maybe a note under the Create and Edit a Scope sections that says the following:

`An OAuth scope is implicitly enabled once it has been created, allowing the application to include that scope on an OAuth request.`

Reply:
Works for me.